### ISSUE TITLE
Limit the number of displayed errors by default

### ISSUE THREAD

**Issue by artlogic** [arsmentis/rusnack-pd#10]
Only list the first 3 or 5. Hide the rest by default. Allow the user to expand the display to the full number of errors.

**Comment by SKBruck** [comment]
This is partly possible with bootstrap's collapse component, and I can code the example, but I feel like this is going to require counting and assigning a special class to error messages after 5.

**Comment by artlogic** [comment]
I can take care of that - just get the example going.

**Comment by SKBruck** [comment]
Example is pushed. Let me know if that adding margin-bottom is going to be an annoyance.

**Comment by SKBruck** [comment]
See More button wasn't styled responsively, so I'll need to fix that once this issue has been reviewed.

**Comment by SKBruck** [comment]
I agree with both points - plus the color currently matches the Remove button color, which is not accessible.

Also, I think the Show More... button should match the length of the Remove button on xs widths. That's the styling issue I mentioned.

**Comment by artlogic** [comment]
The only other thing is that the show more / show less button should probably always be at the bottom of the content - rather than in the middle when expanded

**Comment by SKBruck** [comment]
Please review the changes I pushed - style changed to match, I moved the button below the errors, and I made the button text toggle. If you like it, please close the issue.

**Comment by artlogic** [comment]
This is looking really solid.

I'm wondering how you feel about the remove button and the show less/more button when at mobile widths? It feels a bit odd to me.

**Comment by SKBruck** [comment]
I prefer full width tap targets on small screens because it's most accessible, but I know it's not the prettiest solution. What's feeling odd to you?
